packages/terra-collapsible-menu-view/src/CollapsibleMenuViewItemGroup.jsx

```diff
@@ -80,7 +80,7 @@ class CollapsibleMenuViewItemGroup extends React.Component {
     } if (isCollapsibleMenuItem) {
       return (
         <li role="none">
-          <List {...customProps} role="listbox">
+          <List {...customProps} role="menu">
```

Comment:
I think listbox is the role we want here. According to https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role: 
> The listbox role is used to identify an element that creates a list from which a user may select one or more static item

This example even uses the role `listbox` for a collapsible menu:
https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-collapsible.html

Reply:
Actually, I think the `li` and `List` can just be removed entirely for this scenario and only return children. This code block is to build out an un-selectable CollapsibleMenuViewItemGroup there's functionally no need to have the items wrapped by anything when displayed in the Menu.

Reply:
Removed in 8dd92b0